reactor-core/src/main/java/reactor/util/concurrent/MpscLinkedQueue.java

```diff
@@ -28,6 +24,10 @@
 
 import reactor.util.annotation.Nullable;
 
+/*
+ * The code was inspired by the similarly named JCTools class:
```

Comment:
this was moved otherwise spotlessApply would erase it (if before the `package` statement)